os/logm/logm.h

```diff
@@ -72,16 +71,16 @@ extern "C" {
 #define EXTERN extern
 #endif
 
-EXTERN volatile int g_logm_head;
-EXTERN volatile int g_logm_tail;
-EXTERN volatile int g_logm_available;
-EXTERN volatile int g_logm_enqueued_count;
-EXTERN volatile int g_logm_overflow_offset;
-EXTERN volatile int g_logm_dropmsg_count;
-EXTERN char *volatile g_logm_rsvbuf;
-EXTERN volatile int logm_bufsize;
+EXTERN int g_logm_head;
```

Comment:
Is there any specific reason for EXTERN type?
As possible as you can, don't use extern type to maintain code easily.
If other file has to access these file, you'd better make function instead of direct accessing.